R/read_redcap_tidy.R

```diff
@@ -2,32 +2,32 @@
 #' Extract a REDCap database into a tidy supertibble
 #'
 #' @description
-#' Call the REDCap API to retrieve data and metadata about a project,
-#' and then transform the output into a tidy "supertibble" that contains data
-#' and metadata organized into tidy tibbles, broken down by instrument.
+#' Query the REDCap API to retrieve data and metadata about a project,
+#' and transform the output into a "supertibble" that contains data
+#' and metadata organized into tibbles, broken down by instrument.
 #'
 #' @details
 #' This function uses the [REDCapR](https://ouhscbbmc.github.io/REDCapR/index.html)
-#' package to call the REDCap API. The REDCap API returns a
+#' package to query the REDCap API. The REDCap API returns a
 #' [block matrix](https://en.wikipedia.org/wiki/Block_matrix) that mashes
 #' data from all data collection instruments
-#' together. The `read_redcap_tidy()` function intelligently
+#' together. The `read_redcap_tidy()` function
 #' deconstructs the block matrix and splices the data into individual tibbles,
 #' where one tibble represents the data from one instrument.
 #'
 #' @return
-#' Returns a `tibble` in which each row represents a REDCap instrument. The
-#' results contains the following fields:
+#' Returns a `tibble` in which each row represents a REDCap instrument. It
```

Comment:
Looks like elsewhere in the docs we leave the "returns" implicit so we should be consistent here
```suggestion
#' A `tibble` in which each row represents a REDCap instrument. It
```